tests/model_hub_tests/performance_tests/test_tf_hub_perfomance_model.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import os
+import shutil
+
+import gc
+import tarfile
+import tempfile
+import pytest
+import tensorflow as tf
+import openvino as ov
+import tensorflow_hub as hub
+# noinspection PyUnresolvedReferences
+import tensorflow_text  # do not delete, needed for text models
+import numpy as np
+import json
+
+from models_hub_common.test_performance_model import TestModelPerformance
+from models_hub_common.utils import get_models_list
+from models_hub_common.constants import tf_hub_cache_dir
+from models_hub_common.constants import no_clean_cache_dir
+
+
+def clean_cache():
+    if not os.path.exists(tf_hub_cache_dir):
+        return
+    for file_name in os.listdir(tf_hub_cache_dir):
+        file_path = os.path.join(tf_hub_cache_dir, file_name)
+        try:
+            if os.path.isfile(file_path) or os.path.islink(file_path):
+                os.unlink(file_path)
+            elif os.path.isdir(file_path):
+                shutil.rmtree(file_path)
+        except Exception as e:
+            pass
+
+
+class DownloadInfo:
+    MODEL_LINK_KEY = 'model_link'
+    PATH_KEY = 'path'
+
+    def __init__(self, line: str = None):
+        self.model_link = ''
+        self.path = ''
+        self.untar_path = ''
+        if line:
+            self.unpack(line)
+
+    def unpack(self, s: str) -> None:
+        item = json.loads(s)
+        self.model_link = item[DownloadInfo.MODEL_LINK_KEY]
+        self.path = item[DownloadInfo.PATH_KEY]
+
+    def pack(self) -> str:
+        item = dict()
+        item[DownloadInfo.PATH_KEY] = self.path
+        item[DownloadInfo.MODEL_LINK_KEY] = self.model_link
+        return json.dumps(item)
+
+
+DOWNLOADED_FILES_LIST_NAME = 'downloaded_files.txt'
+
+
+def get_model_downloaded_info(model_link):
+    path = os.path.join(tf_hub_cache_dir, DOWNLOADED_FILES_LIST_NAME)
+    if not os.path.exists(path):
+        return False
+    try:
+        file_lines = []
+        with open(path, 'r') as f:
+            file_lines = f.readlines()
+        for line in file_lines:
+            item = DownloadInfo(line)
+            if item.model_link == model_link:
+                return item
+    except json.JSONDecodeError as e:
+        print('downloaded files list is corrupted: remove cache')
+        clean_cache()
+        return None
+    return None
+
+
+def save_model_download_info(new_download_info):
+    download_list_path = os.path.join(tf_hub_cache_dir, DOWNLOADED_FILES_LIST_NAME)
+    with open(download_list_path, 'a') as f:
+        f.write(new_download_info.pack())
+        f.write('\n')
+
+
+def download_model_from_tf_hub(model_name: str, model_link: str):
+    return hub.resolve(model_link)
+
+
+def get_model_list_path(filename: str) -> str:
+    parent_dir = os.path.dirname(os.path.dirname(__file__))
+    return os.path.join(parent_dir, 'tf_hub_tests', filename)
```

Comment:
Now you use `precommit_models` from `tf_hub_tests` directory but it should be from `performance_tests`

Reply:
it is only needed for nightly tests, that use the same model list as tf_hub_tests. Currently we remove them from performance tests, so I will delete it